pkgs/development/libraries/qt-6/default.nix

```diff
@@ -185,10 +185,31 @@ let
       };
 
       wrapQtAppsHook = callPackage
-        ({ makeBinaryWrapper }: makeSetupHook
+        ({ makeBinaryWrapper, runCommand, qtwayland, qtbase }:
+          let
+            onlyPlugins = drv: (runCommand "${drv.__spliced.targetTarget.name or drv.name}-only-plugins" { } ''
+                mkdir -p "$out/lib/qt-6"
+                ln -s "${drv.__spliced.targetTarget or drv}/lib/qt-6/plugins" "$out/lib/qt-6/plugins"
+                ln -s "${drv.__spliced.targetTarget or drv}/lib/qt-6/qml" "$out/lib/qt-6/plugins"
```

Comment:
Wrong path here, can also use `qtbase.qtPluginPrefix`/`qtbase.qtQmlPrefix` to avoid duplicating.